manifests/agent.pp

```diff
@@ -414,9 +416,9 @@
   # https://support.zabbix.com/browse/ZBX-11631
   if $facts['selinux'] == true and $manage_selinux {
     selinux::module{'zabbix-agent':
-      ensure    => 'present',
-      source_te => 'puppet:///modules/zabbix/zabbix-agent.te',
-      before    => Service['zabbix-agent'],
+      ensure     => 'present',
+      content_te => template('zabbix/selinux/zabbix-agent.te.erb'),
```

Comment:
@tux-o-matic What I meant is that zabbix/selinux/zabbix-agent.te.erb should also should be the default value of a new parameter named say $selinux_content_te , so if a user doesn't want to use the selinux template included with this module, they can use their own as well.

Reply:
The template is an empty shell, everything is doable via the new class parameters. This PR only adds flexibility.